packages/platform/atoms/vite.config.ts

```diff
@@ -52,6 +52,7 @@ export default defineConfig(({ mode }) => {
         path: resolve("../../../node_modules/rollup-plugin-node-builtins"),
         os: resolve("../../../node_modules/rollup-plugin-node-builtins"),
         "@": path.resolve(__dirname, "./src"),
+        "@calcom/lib/markdownToSafeHTML": "@calcom/atoms/monorepo",
```

Comment:
this ensures that platform atoms are not using the markdownToSafeHTML from web but our custom implementation